### DISCUSSION TITLE
How do I "stop" auto-merge without closing the PR?

### DISCUSSION THREAD

**Discussion by klippx** [renovatebot/renovate#31636] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub / `renovatebot/github-action@v40.2.10`

### Please tell us more about your question or problem

As you can see in screenshots below, I saw this PR and wanted it NOT merged. But also NOT closed, since I don't want to forget about it. So I tried to disable auto-merge. But the next run renovate enabled auto merge again, in the run after it merged it. 

Is there any configuration I can do to get it to NOT enable auto merge on a PR that I explicitly want to "pause"?

<img width="928" alt="Screenshot 2024-09-26 at 9 37 18 AM" src="https://github.com/user-attachments/assets/c215f541-9290-44ff-b0a3-959ddc481a2a">
<img width="957" alt="Screenshot 2024-09-26 at 9 37 29 AM" src="https://github.com/user-attachments/assets/29631e0f-c51a-4ac0-9651-089d0ccb3626">

Config:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
  "labels": ["dependencies"],
  "extends": ["config:recommended", ":disableDependencyDashboard"],
  "prHourlyLimit": 2,
  "prConcurrentLimit": 16,
  "platformAutomerge": true,
  "commitMessageSuffix": "`[skip cd]`",
  "packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "commitMessageSuffix": null,
      "automerge": false
    },
    {
      "matchUpdateTypes": ["patch", "minor"],
      "matchCurrentVersion": "!/^0/",
      "automerge": true
    },
    {
      "matchUpdateTypes": ["pin", "digest"],
      "automerge": true
    }
  ],
  "lockFileMaintenance": {
    "enabled": true,
    "automerge": true
  }
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

I did not run this with debug logs on, and did not reproduce it. LMK if this is required.

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
There isn't a official way to do this. But, you could modify the PR, as Renovate doesn't automerge PRs that are modified.

**Reply by klippx** [reply]
Tried updating the title, but the bot edits it back... 😢 

<img width="997" alt="Screenshot 2024-09-26 at 3 51 03 PM" src="https://github.com/user-attachments/assets/b5d27cf6-118f-4a40-b8b7-992d84db8dc7">

**Reply by rarkins** [reply]
you need to push a commit
